app/src/main/res/layout/tab_tray_item.xml

```diff
@@ -57,15 +57,18 @@
         android:layout_height="wrap_content"
```

Comment:
This is not advisable to be used with auto sizing.

Reply:
Do you have a suggestion what should be used as `layout_height` value instead?

Reply:
If UX approves using auto sizing, they should also specify a fixed dp height for this in the specs.

Reply:
I will wait for UX to weight in then